src/demo.rs

```diff
@@ -689,15 +689,36 @@ fn handle_packet(packet: &Packet, state: &GameState) -> Vec<DemoMessage> {
 
 pub struct PrintVotes {
     votes: HashMap<u32, Vec<String>>,
+    shunted_messages: Vec<(u32, DemoMessage)>,
```

Comment:
I'd consider making this a `Vec<(u32, VoteCastEvent, Option<SteamID>)>` instead so that you can ensure they are the only type that will be dumped into the vector and you won't have to pattern match to get the vote event back out again later.

Reply:
eh i had it like this because i wanted solution that could be used for any message we might want to shunt, but eh ive just renamed it to `shunted_vote_cast_message` and done as suggested

Reply:
Ah yeah that is a reasonable justification, although I think for the purpose of this handler it probably won't need to do this with any other ones in the future, and if it does it should be a pretty easy change to make.